content/terraform-docs-common/redirects.jsonc

```diff
@@ -245,43 +245,48 @@
   // Policy enforcement changes related to pre-written Sentinel policies
   {
     "source": "/terraform/cloud-docs/policy-enforcement/sentinel",
-    "destination": "/terraform/cloud-docs/policy-enforcement/define-policies/sentinel",
+    "destination": "/terraform/cloud-docs/policy-enforcement/define-policies/custom-sentinel",
```

Comment:
This redirect URL needed to be fixed, the URL is actually `/terraform/cloud-docs/policy-enforcement/define-policies/custom-sentinel`